client/blocks/login/two-factor-authentication/push-notification-illustration.jsx

```diff
@@ -2,7 +2,7 @@
  * External dependencies
  */
 import React from 'react';
-import { colors as PALETTE } from '@automattic/color-studio';
+import colorStudio from '@automattic/color-studio';
```

Comment:
https://webpack.js.org/migrate/5/#cleanup-the-code
https://webpack.js.org/blog/2020-10-10-webpack-5-release/#json-modules

JSON modules don't export the properties anymore, only a default property